### DISCUSSION TITLE
8.8.0 Progress

### DISCUSSION THREAD

**Discussion by wasabii** [ikvmnet/ikvm#496] [Announcements]
The changes currently staged in develop, slated for 8.8, are:

* Use System.Reflection.Metadata to write assemblies in IKVM.Reflection.
* Enable win-arm64 Framework tools and images
* javax.media.sound: should enable MIDI support. Currently broken on OS X due to some issues in the native code.
* Remove DllExport
* IKVM.Reflection ILGenerator/PDB Rewrite: we now generate only portable PDBs. This enables debugging on CoreCLR.
* Upgrade to JDK8u152

**Comment by GeorgeS2019** [comment]
> Remove DllExport

This sounds like a **major milestone**.  It would be great if we get a little reply to learn why :-)
